src/Login.js

```diff
@@ -1,18 +1,49 @@
 import React, { Component } from 'react';
 import MockData from './mockData.js';
 import App from './App.js';
+import BarLogo from './BarStock_Logo.png'
 import './css/Login.css';
 
-const Login = () => {
-  return (
-    <div className="Login">
-      <h2>Login</h2>
-    </div>
-  );
-};
 
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userNameValue : "",
+
+    }
+  }
+
+  handleClick = (e) => {
+    e.preventDefault();
+    this.props.toggleLogin();
+    let userName = document.getElementById("userNameID").value;
```

Comment:
Accessing this element directly is bypassing the built-in event system for React - which is something you shouldn't do unless absolutely necessary (and it's not necessary in this case). If you're looking to get the value from your input, this is something that you can get from the event directly. [This section](https://reactjs.org/docs/forms.html#controlled-components) of the React docs gives a good example of setting up a controlled form.